### ISSUE TITLE
Cloth component > Simulate in editor: Does not handle colliders

### ISSUE THREAD

**Issue by ohmaya** [o3de/o3de#17027]
While Run > Simulate does take colliders into consideration.

Thanks.

**Comment by adamdbrw** [comment]
Could you describe this issue in terms of observed vs expected behaviors?

**Comment by ohmaya** [comment]
Basically, the difference:
- For EditorClothComponent, ActorComponentRequests::GetPhysicsConfig() is called.
- For ClothComponent, ActorComponent::GetPhysicsConfig() is called.

**Comment by michalpelka** [comment]
@ohmaya thank you for the feedback. You could explain in detail the behavior of your scene in the context of simulation and running in game mode.
The PhysX scene in the Editor is simpler to run in game mode (or game launcher). You can see a difference in those scenes with PhysX Nvidia Debugger. It is possible that the behavior for `ClothComponent` and `EditorClothComponent` is different because some PhysX primitives are not created or set up in the scene in Editor. 
You can verify if there is no blocker for that from the simplification mentioned above. If true feel free to request such a feature.

**Comment by michalpelka** [comment]
Agreed in sig-simulation issue triage, I added `triage/needs-information` label, and we are waiting for 7.12.2023. 
If no further information is provided, I will close the issue.

**Comment by ohmaya** [comment]
already.. fixed & **solution provided**.. 3 weeks ago..

**Comment by michalpelka** [comment]
Great, thanks @ohmaya! Please provide a link to PR with the solution mentioned above, since I cannot find it.

**Comment by michalpelka** [comment]
@ohmaya it would be great to provide a PR to fix your issue since you stated that was fixed. I do not have much experience with NvCloth Gem, but we would be more than happy to review your contribution merge its development branch, and include it in the next releases.
Here is a guide on contribution to O3DE: https://www.docs.o3de.org/docs/contributing/ .

**Comment by ohmaya** [comment]
Setting up contribution looks involving quite something to go through.

While implementing the feature is easy:
- The 3 virtual functions are called in ActorClothColliders.cpp by ActorComponentRequestBus::EventResult().
- Just copy the implementation of those 3 fuctions from ActorComponent to EditorActorComponent, then Simulate in editor" will take colliders into consideration.
